cl/monitor/metrics.go

```diff
@@ -121,6 +122,11 @@ func microToMilli(micros int64) float64 {
 	return float64(micros) / 1000
 }
 
+// ObserveNumberOfAggregateAndProofSignatures sets numbeer of signatures in aggregate and proof
+func ObserveNumberOfAggregateAndProofSignatures(signatures int) {
+	aggregateAndProofSignatures.Set(float64(signatures))
```

Comment:
please collect a per-aggregate average here see how the `signature verification amortized` metric works

Reply:
@Giulio2002 i'm not 100% sure on what is right way to monitor in your case.  
But if use `Summary` instead of `Counter` then it will automatically provide: quantiles (on timings) and count amount of events/sec. 
You will not need calc averages by self.

Example: 
```
// create summary
var myMetric = GetOrCreateSummary(`kv_get{level="L0",domain="account"}`)

// add raw data
func (dt *DomainRoTx) getLatestFromFile() {
	defer myMetric.ObserveDuration(time.Now())
	
        // ...
}

// can create Grafana chart on `timings` 
kv_get{quantile="$quantile",instance=~"$instance"}

// can create Grafana chart on `events/sec`
rate(kv_get_count{instance=~"$instance"}[$rate_interval])
``` 

<img width="2027" alt="Screenshot 2024-10-22 at 13 04 04" src="https://github.com/user-attachments/assets/40fbba40-d890-4ae1-af09-a0112588e3b8">


<img width="2031" alt="Screenshot 2024-10-22 at 13 06 59" src="https://github.com/user-attachments/assets/cf1de247-5658-4713-aa04-698e0921ba09">

<img width="2030" alt="Screenshot 2024-10-22 at 13 08 21" src="https://github.com/user-attachments/assets/18c9a75f-046d-456d-8e44-73af1d9271d7">

Reply:
no i want to collect a moving average